fastlane/Fastfile

```
@@ -0,0 +1,119 @@
+  # returns the highest version code from array
+  return versionCodes.max
+end
+
+
+default_platform(:android)
+platform :android do
+
+  desc "Build and Distribute Debug APK to Firebase"
+  lane :build_and_distribute_debug do |options|
+
+   latest_release = firebase_app_distribution_get_latest_release(
```

rubocop (1.69.1)

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)